frontend/farm_designer/plants/select_plants.tsx

```diff
@@ -53,28 +53,40 @@ export class RawSelectPlants extends React.Component<SelectPlantsProps, {}> {
     }
   }
 
+  SelectedNotUndef = (input?: string[] | undefined) => {
```

Comment:
I am going to change the first letter of this method to lowercase since it is not being used as a constructor / React component.